### ISSUE TITLE
Neo4J browser(?)

### ISSUE THREAD

**Issue by kaplun** [inspirehep/inspire-next#1092]
Cataloger and power users will need a better interface than the current search engine, in order to explore data.

By pushing all relations to Neo4J we could in principle allow those users to navigate the data with a graph-based interface such as [Neo4J Browser](http://neo4j.com/developer/guide-neo4j-browser/).

However, I wonder if we can assume catalogers will be able to express their queries in Cypher? Let's hope so :smile: 

In that case would be great to integrate Neo4J Browser within INSPIRE.

E.g. imagine visiting http://inspirehep.net/relation-browser and be able to navigate and discover information...

**Comment by kaplun** [comment]
@eamonnmag WDYT?

**Comment by aw-bib** [comment]
> However, I wonder if we can assume catalogers will be able to express their queries in Cypher?

I tend to say "no".

**Comment by kaplun** [comment]
To have a more informed discussion here's what I am talking about: 
https://www.youtube.com/watch?v=Go3P73-KV30

**Comment by eamonnmag** [comment]
I don't think this will work very well. I think showing sub networks is potentially nice, which is pretty easy with an [out of the box force directed layout](http://bl.ocks.org/mbostock/4062045) showing author nodes, paper nodes, keywords, etc., making them interactive, etc. I think a nice interface can be made for exploration, but certainly showing everything will not be possible to show using standard graphs.

Showing too many will bring your browser to a stand still (e.g. over 2000 highly connected nodes). There are some other ways of visualizing dense networks in a more scalable way, e.g. hive plots.https://bost.ocks.org/mike/hive/

Hive plots may look a bit confusing at the start, but the basic premise is that authors, experiments, papers, etc. would all be separate axes, and the connections within and between categories are visualised via a typical linked connection. They are also less visually attractive I'd say :)

**Comment by kaplun** [comment]
The thing is that I wonder how to allow catalogers to query INSPIRE for non-anticipated queries exploiting graphs. So I was looking into Neo4J browser as an easier interface to let them build more complex queries.

**Comment by eamonnmag** [comment]
Forgot about this issue! See #1193
